src/sharedm/headers1/headers1.component.css

```diff
@@ -0,0 +1,72 @@
+
+
+@media(min-width:768px) {
```

Comment:
Since we are using bootstrap we will avoid adding media queries, these changes are only valid for screen sizes 768px check responsiveness with chrome tools.